terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/validation/compare.jsp

```diff
@@ -0,0 +1,49 @@
+<h2>@Compare Test</h2>

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...